src/main/java/com/google/sps/data/User.java

```diff
@@ -3,14 +3,25 @@
 import java.util.Collection;
 import java.util.HashSet;
 
+
 public final class User {
+  public static enum profileType {
+    STUDENT, MENTOR;
+  };
 
-  private final String name;
-  private final Collection<String> productArea;
+  private Long id;
```

Comment:
Is there a reason to use the Long type here instead of the primitive long type for the id?

Reply:
Oh, I thought it was Long not long, I'll change it.

Reply:
Done